LayoutKitTests/OverlayLayoutTests.swift

```diff
@@ -111,10 +185,42 @@ class OverlayLayoutTests: XCTestCase {
                 1.0: expectedFrame,
                 2.0: expectedFrame,
                 3.0: expectedFrame
-            ])
+                ])
         }
     }
 
+    /**
+     Tests primary layouts smaller than overlay.
+     Sublayouts should have same measurements.
+     */
+    func testPrimaryLayoutsSmallerThanOverlay() {
+        let primaryLayout0 = SizeLayout<View>(width: 2, height: 2)
+        let primaryLayout1 = SizeLayout<View>(width: 3, height: 4)
+        let sublayout = SizeLayout<View>(width: 10, height: 10)
+
+        let overlay = OverlayLayout(primaryLayouts: [primaryLayout0, primaryLayout1], overlayLayouts: [sublayout])
```

Comment:
I suggest renaming sublayout to overlay or overlayLayout.

Reply:
Renamed to overlayLayout0.